src/components/Button.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../Button.css";
+export default function Button(props) {
```

Comment:
SPACE please :) 

Also instead of using the `function` keyword it is better we use arrow functions.